src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+
+const convertToNumber = str => Number(str.replace(/[^\d.-]/g, ''));
+
+const sortList = (list, sortParam) =>
+  [...list].sort(
+    (a, b) =>
+      convertToNumber(b.dataset[sortParam]) -
+      convertToNumber(a.dataset[sortParam]),
+  );
+
+const getEmployees = (sortedList, element) => {
+  element.innerHTML = '';
+
+  sortedList.forEach(item => {
+    const li = document.createElement('li');
+
+    li.setAttribute('data-position', item.dataset.position);
+    li.setAttribute('data-salary', item.dataset.salary);
+    li.setAttribute('data-age', item.dataset.age);
+
+    li.textContent = item.innerText;
+
+    element.appendChild(li);
+  });
+};
+
+const visibleEmployees = sortList([...listEl.children], 'salary');
+
+getEmployees(visibleEmployees, listEl);
```

Comment:
You're calling `getEmployees` with `visibleEmployees` and `listEl`. This is fine, but it's not clear why you're passing `listEl` as a parameter. Since `listEl` is defined in the global scope, you could just use it directly in the `getEmployees` function. If you want to keep `getEmployees` flexible and not dependent on the global scope, consider passing the HTML string of the sorted list items to `getEmployees` instead of passing the sorted list of elements and the list element.